impl/src/main/java/org/ehcache/spi/copy/DefaultCopyProvider.java

```diff
@@ -58,20 +55,24 @@ public DefaultCopyProvider(DefaultCopyProviderConfiguration configuration) {
   private <T> Copier<T> createCopier(CopierConfiguration.Type type, Class<T> clazz,
                                      Serializer<T> serializer, ServiceConfiguration<?>... configs) {
     DefaultCopierConfiguration<T> conf = find(type, configs);
-    if(conf != null && conf.getClazz().isAssignableFrom(SerializingCopier.class)) {
-      return new SerializingCopier<T>(serializer);
+    Copier<T> copier;
+    if (conf != null && conf.getInstance() != null) {
+      copier = conf.getInstance();
+    } else if (conf != null && conf.getClazz().isAssignableFrom(SerializingCopier.class)) {
+      copier = new SerializingCopier<T>(serializer);
+    } else {
+      copier = createCopier(clazz, conf);
     }
-    return createCopier(clazz, conf);
+    LOG.info("Copier for <{}> : {}", clazz.getName(), copier);
+    return copier;
   }
 
   private <T> Copier<T> createCopier(Class<T> clazz, DefaultCopierConfiguration<T> config) {
     String alias = (config != null ? null : clazz.getName());
     Copier<T> copier = (Copier<T>) newInstance(alias, config);
     if (copier == null) {
-      LOG.info("No registered copier found for for <{}>. Using the default Identity copier.", clazz.getName());
```

Comment:
That particular location wasn't ideal, so I've moved that INFO log a few lines above which makes the logger output less confusing.